patroni/config.py

```diff
@@ -331,7 +334,7 @@ def _process_postgresql_parameters(parameters, is_local=False):
                 or not is_local and ConfigHandler.CMDLINE_OPTIONS[name][1](value)}
 
     def _safe_copy_dynamic_configuration(self, dynamic_configuration):
-        config = deepcopy(self.__DEFAULT_CONFIG)
+        config = deepcopy(self.get_default_config())
```

Comment:
It already returns a copy.
```suggestion
        config = self.get_default_config()
```